yogo/packages/core/src/app.e2e.test.ts

```diff
@@ -0,0 +1,125 @@
+import { bootstrap } from "./bootstrap";
+import { ApolloServer } from "apollo-server";
+import {
+  createTestClient,
+  ApolloServerTestClient
+} from "apollo-server-testing";
+
+describe("e2e", () => {
+  let server: ApolloServer;
+  let url: string;
+  let client: ApolloServerTestClient | null = null;
+
+  beforeAll(async () => {
+    console.log("bootstrapping a server");
+    ({ server, url } = await bootstrap());
+    client = createTestClient(server);
+  });
+  afterAll(async () => {
+    console.log("shutting down the server");
+    await server.stop();
+    client = null;
+  });
+  it("server is defined", () => {
+    expect(server).toBeDefined();
+    expect(url).toBe("http://localhost:4000/");
+  });
+  it("client is defined", () => {
+    expect(client).toBeDefined();
+  });
+
+  describe("Group", () => {
+    it("creates groups", async () => {
+      const res = await client?.query({
+        query: `
+        mutation {
+          createGroup(id: "my-group"){
```

Comment:
The current implementation of the `createGroup` mutation takes `id` as an argument.

While this `id` is supposed to be unique, i.e. the uniqueness of the id must be validated on the server-side, we could probably just depend on uuid/v4 or something for now